app/actions/remote/channel.ts

```diff
@@ -220,7 +220,7 @@ export const fetchMyChannelsForTeam = async (serverUrl: string, teamId: string,
 
         if (!fetchOnly) {
             storeMyChannelsForTeam(serverUrl, teamId, channels, memberships);
-            storeCategories(serverUrl, categories);
+            storeCategories(serverUrl, categories, false, true); // Re-sync
```

Comment:
wooooowwww I would be careful here... Have you checked what triggers this `fetchMyChannelsForTeam` function? you should probably use an optional parameter

Reply:
Yup - so the different places this is called and channels are fetched, it seems appropriate to also double check that the categories are also in sync; especially from @larkox's comment around websockets potentially not being available.

Reply:
So your plan is that every time we fetch channels we sync the entire categories models?

Reply:
It was already doing this? All `prune` adds is deleting categories that no longer exist.

Unless there's a better way? Other option is to go back to having category fetching / syncing outside of channels and calling it when needed 🤷